js/about.js

```diff
@@ -1,4 +1,6 @@
 const menu = document.querySelector('.open');
 menu.addEventListener('click', () => {document.getElementById('mobile-menu').classList.add('active') })
 const close = document.querySelector('.close');
-close.addEventListener('click', () => {document.getElementById('mobile-menu').classList.remove('active')})
\ No newline at end of file
+close.addEventListener('click', () => {document.getElementById('mobile-menu').classList.remove('active')})
+const ulClose = document.querySelector('.ul-close');
+ulClose.addEventListener('click', () => {document.getElementById('mobile-menu').classList.remove('active')})
```

Comment:
- [ ] The mobile size contains an hamburger menu on the left side, nice you have it, however, the hamburger menu should disappear once a user clicks on it and appear the close button on the right side.

Reply:
This issue is well resolved! ✔️